openjdk.test.jck/src/test.jck/net/adoptopenjdk/stf/Jck.java

```diff
@@ -324,6 +324,15 @@ public void setUp(StfCoreExtension test) throws Exception {
 		
 		test.doEchoFile("The JTB file generated\n", newJtbFileRef);
 
+		if ( tests.contains("api/javax_net") ) {
```

Comment:
Just wondering, do these changes require any set up to be done somewhere else for them to work?